tests/schema/invalid_container_with_wrong_declared_envVars.json

```diff
@@ -0,0 +1,13 @@
+{
+    "$schema": "../../deeployer.schema.json",
+    "containers": {
+        "elasticsearch": {
+            "image": "",
+            "replicas": "1",
+            "ports": "",
+            
+            "env" : " app : mysql "
```

Comment:
This test is good.

You should also add a test testing env variables with a special object like this:

```
"env": {
  "FOO": { "SOME": "SUBOBJECT" }
}
```

This should fail too.

And possibly, add another one to test that all keys are strings (integers and booleans are forbidden):

```
"env": {
  "FOO": 42
}
```
This should fail too.